docs/src/operations/requirements.md

```diff
@@ -55,28 +63,18 @@ made.
 - CPU
   - 16 cores / 32 threads, or more
 - RAM
-  - 512 GB or more if `account-index` is used
+  - **1.17** 512 GB or more if an `account-index` is used, 1TB+ for all three [account indexes](https://docs.solanalabs.com/operations/setup-an-rpc-node#account-indexing)
+  - **1.18 or newer** 512 GB for all three indexes
```

Comment:
I don't personally have access to a 512 GB node, but might be nice to get some confirmation that 512 GB nodes run ok.

I'll see if I can track one down to test myself or ask some people who are likely running; we had some previous reports of people seeing significant memory reduction with v1.18 but it isn't clear to me if they had > 512 GB machines but saw utilization below 512 GB. I think this is a useful distinction to make as a 768 / 1024 GB node has a lot of headroom to ride out any temporary spikes.

Could also ask around on Discord too

Reply:
I ran it here for a while, seemed to stick around 300GB in RAM:
https://x.com/ferric/status/1793394216732618862

Had someone else confirm 150G usage wiwth all three indexes

but now i'm hearing a conflicting report:
https://discord.com/channels/428295358100013066/837340113067049050/1244844350567878767

Will ask around more

Reply:
positive report:
https://discord.com/channels/428295358100013066/837340113067049050/1245647871181586453

300GB across all three indexes, same as I was seeing

Reply:
another report:
https://discordapp.com/channels/428295358100013066/837340113067049050/1245875650397733016

looks like this person is running all three indexes and ALL of accounts (266gb) in RAM. 

Maybe we should say 512GB+ for all 3 indexes on 1.18?

Reply:
> Maybe we should say 512GB+ for all 3 indexes on 1.18?

Yeah, the rest of the lines that mention RAM say "X GB or more`. So yes, I agree; let's add "512 GB or more" to make this consistent with the others.

I was able to scrounge a node with 512 GB RAM; however, it is only an `AMD 7502P (32 core)`. I disabled 8 of the cores which works ok enough for several other of our non-RPC nodes. But, my node is still struggling to keep up with the cluster.

Let me fiddle with it a bit more, I'd like to be able to say that "we" (Anza) have officially reproduced running all 3 indexes with 512 GB RAM. That being said, I know I'm running with not great CPU, so if I can't get it working in another day or two, I'll admit defeat with this HW and we'll justify the change with the community reports.

Thank you for your continued patience on this. The past two weeks have been very "irregular" in regards to workdays, and I've had much less time than normal in front of my computer to do "normal work". This is a normal week so responses will be much more timely

Reply:
Updated to 512GB or more in 8a1c471cf9